src/app-layout/__tests__/common.test.tsx

```diff
@@ -286,15 +287,15 @@ describeEachAppLayout(size => {
         },
       };
 
-      const { wrapper } = renderComponent(<AppLayout contentType="form" {...drawersOpen} />);
+      const { wrapper } = renderComponent(<AppLayout contentType="form" {...(drawersOpen as any)} />);
 
       wrapper.findActiveDrawerCloseButton()!.click();
       expect(onChange).toHaveBeenCalledTimes(1);
       expect(onChange).toHaveBeenCalledWith(expect.objectContaining({ detail: null }));
     });
 
     test('Renders aria-expanded only on toggle', () => {
-      const { wrapper } = renderComponent(<AppLayout contentType="form" {...singleDrawer} />);
+      const { wrapper } = renderComponent(<AppLayout drawers={singleDrawerPublic} />);
```

Comment:
Why is `contentType="form"` gone now?

That was a workaround for typescript, because solo spread `<AppLayout {...singleDrawer}>` thrown a type error. So we added a dummy prop (could be any valid public prop) to make typescript accept the spread value as well